htdocs/projet/tasks/time.php

```diff
@@ -418,7 +418,8 @@
 		$tmptimespent = new Task($db);
 		$tmpproduct = new Product($db);
 		$fuser = new User($db);
-
+		$remiseproject = GETPOST('remiseproject', 'int');
```

Comment:
The data is a string used by user to enter a numeric value;
So you must use here
$remiseproject = price2num(GETPOST('remiseproject', 'int'));

Reply:
Done. Is it possible to combine both of your comments and change this to:
$remiseproject = price2num(GETPOSTINT('remiseproject'));?

Thanks for your answers. It is my first Request. :-)

Reply:
GETPOSTINT will always return an int, so no need to use price2num.
But an amount or rate is not an int, it is a float with a possible decimalseparator. So you must use GETPOST and then convert it into a float according to the currency format of the user with price2num()

Reply:
OK. Done. Thanks a lot